syntax/blocks.js

```diff
@@ -239,28 +243,45 @@ var english = {
 
   dropdowns: {},
 
-  commands: {},
+  commands: [],
```

Comment:
I think this should still be an object? If I understand correctly, its values are now arrays, but it’s keys are still strings.

Reply:
Whoops